actions/tomee-dependency/main.go

```diff
@@ -66,9 +66,27 @@ func main() {
 		panic(err)
 	}
 
+	latestVersion, err := versions.GetLatestVersion(inputs)
+	if err != nil {
+		panic(err)
+	}
+
 	if o, err := versions.GetLatest(inputs); err != nil {
 		panic(err)
 	} else {
+		// override the version with the full string (including prerelease information)
+		if latestVersion.Prerelease() != "" {
+			o["version"] = latestVersion.String()
+		}
 		o.Write(os.Stdout)
 	}
```

Comment:
```suggestion
	latestVersion, err := versions.GetLatestVersion(inputs)
	if err != nil {
		panic(err)
	}

	o, err := actions.NewOutputs(versions[latestVersion.Original()], latestVersion, nil)
	if err != nil {
		panic(err)
	}

	// override the version with the full string (including prerelease information)
	if latestVersion.Prerelease() != "" {
		o["version"] = latestVersion.String()
	}
	o.Write(os.Stdout)
```

It seems unnecessary to call `versions.GetLatest`. That method is doing two things. Line 69-72 is doing the first one of those things, selecting the latest version. The other part is making the outputs. It might be more direct if we just make the output items inline here and skip calling `versions.GetLatest` altogether.